src/App.tsx

```diff
@@ -1,21 +1,28 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+// import styled from '@emotion/styled' // if you want to use styled components, this is in the packages
+
+const IMAGES_URL = "https://dog.ceo/api/breed/hound/images/random/55";
+/**
+ *
+ * @returns { "message": ["https://images.dog.ceo/breeds/hound-afghan/n02088094_1126.jpg", ...] }
+ *
+ * Step 1: Render all the images returned
+ * Step 2: Create chunks of 10 images from the returned list
+ * Step 3: Render the first 10 of the list
+ * Step 4: Build a way to increment to the next 10
+ * Step 5: Build a way to decrement to the previous 10
+ * Step 6: Create a UI that shows which page you're viewing
+ * Step 7: Write a test that covers a user interaction
+ *
```

Comment:
Maybe we could add some bonus points here that we'd kind of expect a senior/higher level of an engineer to complete? like maybe create a way to search, or fetching past 55 to create a kind of fake server pagination, or idk maybe this is enough.

Reply:
I am inclined to stay on the light end here and see how things turn out for the first few.  We should come up with a couple of extension tasks to ask during interviews though and maybe roll them if we feel we need to expand the scope of the take home.